tripy/nvtripy/utils/ast.py

```diff
@@ -123,8 +122,9 @@ def get_arg_candidate_column_offsets(
             if is_kwarg:
                 arg_node = node.keywords[index - num_positional]
             else:
-                # For methods, the `self` argument is omitted from ast.Call.args
-                if "self" in arg_names:
+                # Detect method calls by examining AST structure
```

Comment:
nit: Might still be useful to keep the old comment to explain why we decrement the index